api/proto/relay/README.md

```diff
@@ -0,0 +1,15 @@
+# Should I use the v2 API?
```

Comment:
There's no indication `api/proto/relay` is part of v2. Maybe this README should be specific to relay or say something like relay is only applicable with v2 architecture?

Reply:
Added a note that the code in the `relay` directory is experimental.